### DISCUSSION TITLE
How to use with existing bottle?

### DISCUSSION THREAD

**Discussion by vesper8** [Whisky-App/Whisky#83] [General]
I already have a bottle, steam is installed, 5 games are installed and all working. I'd like to use Whisky to make it easier to launch these games. It seems Whisky only works if you create a new bottle. Can I edit a configuration file somewhere to point Whisky to my existing bottle and existing games?

**Comment by RussianLioN** [comment]
I already have an installed GPTK and Battle.net with fine working Diablo IV. How to use installed wine soft and config with Wiskey?

**Comment by IamWJC** [comment]
You can import under the File Menu of Whisky
